src/restoreNames.test.js

```diff
@@ -1,11 +1,88 @@
 'use strict';
 
 describe('restoreNames', () => {
-  // const { restoreNames } = require('./restoreNames');
+  const { restoreNames } = require('./restoreNames');
 
-  it('should ', () => {
+  // write tests here
+  it(`should be declared`, () => {
+    expect(restoreNames).toBeInstanceOf(Function);
+  });
 
+  it(`should return nothing`, () => {
```

Comment:
Be more explicit wit descriptions

```suggestion
  it(`should return 'undefined'`, () => {
```